### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by Flamefire**
Old virtualenv uses non-existing pip flags

### CODE REVIEW

**Comment by scls19fr** [comment]
Thanks @Flamefire for this contribution which seems to fix #131
Maybe you can try to add Python 3.7 support also (and so it could also fix #112 )

**Comment by Flamefire** [comment]
I used the existing PR #113. Will see if it passes

File: .travis.yml [hunk 1]
```diff
@@ -20,5 +20,8 @@ matrix:
   - python: "3.6"
     env: TOXENV=py36
 
+  - python: "3.7"
+    env: TOXENV=py36
```
Comment (scls19fr) [inline .travis.yml]: I wonder why we didn't have

```
  - python: "3.7"
    env: TOXENV=py37
```

Comment (Flamefire) [reply .travis.yml]: C&P I guess. Fixed

**Comment by scls19fr** [comment]
https://github.com/travis-ci/travis-ci/issues/9815 seems to deal with Python 3.7 support with Travis.

Python 3.7 release date is June 27, 2018 ... so I thought that it was supported "out-of-the-box" by Travis but it doesn't seems to be the case (using `xenial` seems to still be necessary)

**Comment by scls19fr** [comment]
LGTM
Thanks @Flamefire

**Comment by Flamefire** [comment]
Using xenial is the official way for 3.7: https://docs.travis-ci.com/user/languages/python/#python-37-and-higher

**Pull request merged**